test.php

```diff
@@ -0,0 +1,9 @@
+<?php
+class FooBar {
+	public function test() {
+		global $wgOut;
+		$wgOut->addHTML('czesc');
+
+		return wfMsg('foo');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)